subsys/settings/Kconfig

```diff
@@ -104,3 +110,25 @@ config SETTINGS_FS_MAX_LINES
 	depends on SETTINGS && SETTINGS_FS
 	help
 	  Limit how many items stored in a file before compressing
+
+config SETTINGS_NVS_SECTOR_SIZE
```

Comment:
I would fetch sector size value from flash driver or flash_area API. Maybe keep just a multiplier of physical page size here (for support memories with a small pages by NVS.